operators/Cargo.toml

```diff
@@ -21,7 +21,15 @@ schemars = "0.8.10"
 serde = "1.0.143"
 serde_json = "1.0.83"
 serde_yaml = "0.9.9"
-tokio = {version="1.20.1", features = ["macros", "rt-multi-thread", "time"]}
+
+# The tokio crate version should always match the one used in actix-web.
+# The upgrade controller is deployed as a tokio task, and the actix-web must be able to
+# use the same runtime for its listener and workers.
+# E.g., the tokio version used to build actix-web 4.2.1 is tokio 1.13.1.
+# Ref: https://github.com/actix/actix-web/blob/web-v4.2.1/actix-web/Cargo.toml#L121
+tokio = { version = "1.13.1", features = ["rt-multi-thread", "macros", "time"] }
```

Comment:
hmm this should not be true, only needs to be semver compatible which 1.20 is. In the control-plane we use 1.20 with no issues. What issues did you encounter?

Reply:
That's right. I misread this SO answer and wrote that up --> https://stackoverflow.com/questions/66416174/using-actix-from-a-tokio-app-mixing-actix-webmain-and-tokiomain. Thanks!

Reply:
Tested and verified that this works with 1.20.